src/lwc/platformEventListener/platformEventListener.js

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
Similarly we probably should rename this component "utilPlatformEventListener" to match our newly agreed-upon convention and avoid confusing git in the future?